packages/fx-core/tests/component/generator/generator.test.ts

```diff
@@ -54,6 +54,26 @@ const mockedSampleInfo: SampleConfig = {
   downloadUrl: "https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/test",
 };
 
+// The sample prefix is present in the downloadurl of the external sample
+const mockedExternalSampleConfig = {
+  samples: [
+    {
+      id: "test",
+      onboardDate: new Date(),
+      title: "test-title",
+      shortDescription: "test-sd",
+      fullDescription: "test-fd",
+      types: [],
+      tags: [],
+      time: "",
+      configuration: "test-configuration",
+      suggested: false,
+      gifUrl: "",
+      downloadUrl: "https://github.com/OfficeDev/TeamsFx-Samples/tree/main/sample/test",
```

Comment:
Could we replace the real org and repo name with a mocked value?
For example:
"https://github.com/Org/Repo/tree/main/sample/test"